egs/mobvoi/v1/run_e2e.sh

```diff
@@ -0,0 +1,294 @@
+#!/bin/bash
+
+stage=0
+
+
+. ./cmd.sh
+. ./path.sh
+. utils/parse_options.sh
+
+set -euo pipefail
```

Comment:
I think there should be some explanation of how this differs from run.sh and which one is preferred.